src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -31,16 +33,47 @@
     /** Length of a ansi defined color. */
     private static final int ANSI_PREFIX_LENGTH = 5;
 
-    public static String formatNotebook(String header, Notebook notebook) {
+    /**
+     * Method compiles the ArrayList items and appends the items to a String.
+     *
+     * @param notes ArrayList of notes to obtain note title/tags from
+     * @return noteString StringBuilder containing the notes ready to be printed
+     */
+    public static String formatNotes(String header, ArrayList<Note> notes) {
         String formattedString = "";
-        return formattedString;
+        return getNoteString(header, notes, formattedString);
     }
 
-    public static String formatNotebook(Notebook notebook) {
+    /**
+     * Method compiles the ArrayList items and appends the items to a String.
+     *
+     * @return noteString StringBuilder containing the notes ready to be printed
+     */
+    public static String formatNotes(String pinnedHeader, String unpinnedHeader,
+                                     ArrayList<Note> pinned, ArrayList<Note> unpinned) {
         String formattedString = "";
+        formattedString = getNoteString(pinnedHeader, pinned, formattedString);
+        formattedString = getNoteString(unpinnedHeader, unpinned, formattedString);
         return formattedString;
     }
 
+    private static String getNoteString(String header, ArrayList<Note> notes, String formattedString) {
+        formattedString = formattedString.concat(generatesHeader(header));
+
+        for (Note note: notes) {
+            String colorText = colorize("Title: " + note.getTitle() + " "
+                    + note.getTagsName(), Attribute.BRIGHT_CYAN_TEXT());
+            formattedString = formattedString.concat(colorText);
+
+            int truncatedContentLength = Math.min(note.getContent().length(), MAX_MESSAGE_LENGTH - 50);
+
+            String truncatedContent = note.getContent().substring(0, truncatedContentLength).concat("...");
+            formattedString = formattedString.concat(LS + truncatedContent + LS);
```

Comment:
Why do you use LS? Wont the output have 2 empty lines? and the sides will be broken? i.e.

||  Title ....... ||
<< LS here>>
|| Content...||
<< LS here>>
=========

Reply:
Discussion taken off GitHub